src/next.config.js

```diff
@@ -17,6 +17,8 @@ module.exports = withCSS({
       // Make some environment variables accessible from the client
       new webpack.EnvironmentPlugin({
         OPENCOLLECTIVE_REFERRAL: null,
+        OPENCOLLECTIVE_REDIRECT_PATH: process.env.OPENCOLLECTIVE_REDIRECT_PATH,
+        OPENCOLLECTIVE_BASE_URL: process.env.OPENCOLLECTIVE_BASE_URL,
```

Comment:
We don't need default values, it's the job of the plugin to extract them.

See: https://webpack.js.org/plugins/environment-plugin/